fuzz/Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: help
 
-BINARIES := $(shell sed -n '/^\[\[bin\]\]/,/^$$/ { /name\s*=\s*"\(.*\)"/s//\1/p }' Cargo.toml)
+BINARIES := $(shell awk -F' = ' '/^\[\[bin\]\]/,/^$$/{ if ($$1 == "name") { gsub(/"/, "", $$2); print $$2 } }' Cargo.toml)
```

Comment:
nit: we won't catch cases of odd formats such as `name\t= "foo"` or `name="foo"` as we split per ` = ` (i.e. expect a single whitespace)

Reply:
That's a really good catch! 

I had to torture ChatGPT a little bit more for this one.